From db102a2afd2bf2039408164c5b0f0ad698799b3c Mon Sep 17 00:00:00 2001
Message-Id: <db102a2afd2bf2039408164c5b0f0ad698799b3c.1684255084.git.stefan@agner.ch>
In-Reply-To: <c46f6cb84fb189da723a9932a5195a553762e2b6.1684255084.git.stefan@agner.ch>
References: <c46f6cb84fb189da723a9932a5195a553762e2b6.1684255084.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Tue, 16 May 2023 18:33:33 +0200
Subject: [PATCH] [python] drop unnecessary dependency to pycrypto

PR #21567 introduced an unecessary dependency to pycrypto. From PyPI it
seems that pycrypto is largely unmaintained, and I did not found a use
in the Python bindings or in the Python test.

Also drop the comment about cyryptography and ecdsa usage as the library
is meanwhile used in the core CHIP Python libraries.
---
 src/controller/python/BUILD.gn | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/controller/python/BUILD.gn b/src/controller/python/BUILD.gn
index 424ed433b1..d9cb4e588f 100644
--- a/src/controller/python/BUILD.gn
+++ b/src/controller/python/BUILD.gn
@@ -318,10 +318,7 @@ chip_python_wheel_action("chip-core") {
     "ipdb",
     "deprecation",
     "mobly",
-
-    # Crypto libraries for complex tests and internal Python controller usage
     "cryptography",
-    "pycrypto",
     "ecdsa",
   ]
 
-- 
2.40.1
